cmd/argocd/commands/login_test.go

```diff
@@ -29,3 +33,24 @@ func Test_userDisplayName_sub(t *testing.T) {
 	expectedName := "foo"
 	assert.Equal(t, expectedName, actualName)
 }
+
```

Comment:
Suggestion: Can a test case be added for the non core mode as well?

Reply:
Unfortunately, for non core login, I have to setup a server and start that at localhost. That is the reason they did not have this test case at the very onset. Logout and non-core involves clearing the config files hence we have test cases.